sunpy/net/vso/vso.py

```diff
@@ -308,7 +308,7 @@ def mk_filename(pattern, queryresponserow, resp, url):
         return fname
 
     def fetch(self, query_response, path=None, methods=None, site=None,
-              progress=True, overwrite=False, downloader=None, wait=True):
```

Comment:
The net changes in this PR are in need of less discussion than the CI jobs (apparently) so you might want to put them in their own PR and then we can also backport them easily.

Reply:
This entire PR could be backported easily.